src/fides/api/ops/analytics.py

```diff
@@ -41,14 +40,23 @@ async def send_analytics_event(event: AnalyticsEvent) -> None:
     if CONFIG.user.analytics_opt_out:
         return
     try:
-        await analytics_client._AnalyticsClient__send(  # pylint: disable=protected-access
-            event
+        await analytics_client.send_async(event)
+    except AnalyticsError as err:
+        log.warning("Error sending analytics event: {err}")
+    else:
+        log.info(
+            "Analytics event sent: {event.event} with client id: {analytics_client.client_id}"
         )
```

Comment:
This feels like `DEBUG` level to me (if even). That'll be a lot of logs otherwise... and `event` might contain PII so probably best to exclude entirely.